roles/edpm_pre_adoption_validation/tasks/main.yml

```diff
@@ -29,4 +29,4 @@
 
 - name: Validate systemd-container package
   when: edpm_pre_adoption_validation_machined_enabled
-  ansible.builtin.include_tasks: tuned.yml
+  ansible.builtin.include_tasks: machined.yml
```

Comment:
The deployment fails during validation:
```
K [osp.edpm.edpm_pre_adoption_validation : Fail when systemd-container package is missing] ***
[1;30mtask path: /usr/share/ansible/collections/ansible_collections/osp/edpm/roles/edpm_pre_adoption_validation/tasks/machined.yml:27[0m
Friday 25 October 2024  14:28:48 +0000 (0:00:04.076)       0:00:11.765 ******** 
[0;31mfatal: [standalone]: FAILED! => {[0m
[0;31m    "assertion": "is_systemd_container_package.changed",[0m
[0;31m    "changed": false,[0m
[0;31m    "evaluated_to": false,[0m
[0;31m    "msg": "You must have the systemd-container package installed.\nDo not install it manually as that stops VM instances.\nComplete a minor update of the source cloud and reboot\nyour Nova Compute hypervisors ony by one. To reboot\na hypervisor, firstly live-migrate its VM instances.\n"[0m
[0;31m}[0m
```

https://softwarefactory-project.io/zuul/t/rdoproject.org/build/f6b6c615b1b9474b96b57534e1caa20c/log/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/pre-adoption-validation-openstack-pre-adoption-openstack-4h5qq/logs/pre-adoption-validation-openstack-pre-adoption-openstack.log#810

Reply:
thanks for looking, indeed